roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -142,6 +142,17 @@
     state: present
   become: true
 
+# Exclusive to pgd setup authentication
```

Comment:
Ditto. This actually isn't exclusive to pgd.  It affects everything as long as we allow the default pg_hba.conf file to be created with ident auth.  So we want this for everyone.